Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/SpinBox.h

```diff
@@ -35,10 +38,87 @@ namespace AzQtComponents
         class SpinBoxLineEdit;
     }
 
+    namespace internal
+    {
+        /*
+            A small helper for deferring `setValue' events while a user is editing a textbox-like property.
+            In these cases, any external `setValue' events shouldn't overwrite what the user enters.
+        */
```

Comment:
nit: Could these please use the `//!` convention (can be stacked) instead of using `/* */`